tests/plugins/trim_whitespace.test.ts

```diff
@@ -0,0 +1,97 @@
+import { Model } from "../../src";
+import { InformationNotification } from "../../src/types";
+import { getCellContent } from "../test_helpers";
+import { setCellContent, setSelection } from "../test_helpers/commands_helpers";
+import { getRangeValuesAsMatrix, mountSpreadsheet } from "../test_helpers/helpers";
+
+let model: Model;
```

Comment:
a bit error prone and useless to define the model here if it's neither used in an helper nor in a beforeEach